lib/ansible/modules/windows/win_firewall_profile.ps1

```diff
@@ -0,0 +1,60 @@
+#!powershell
+
+# Copyright : (c) 2019, Simon Lecoq <simon.lecoq@live.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+$ErrorActionPreference = "Stop"
+$firewall_profiles = @("Domain", "Private", "Public")
+$firewall_actions = @("NotConfigured", "", "Allow", "", "Block")
```

Comment:
Why not make it an enum or a hashtable?

Reply:
Used toString() method instead as it already translates options to the correct string value